Cubical/Foundations/Prelude.agda

```diff
@@ -301,6 +306,16 @@ funExt⁻ : {B : A → I → Type ℓ'}
   → ((x : A) → PathP (B x) (f x) (g x))
 funExt⁻ eq x i = eq i x
 
+_≡$_ = funExt⁻
+
+funExtS⁻ : {B : I → Type ℓ'}
```

Comment:
Same questions about this one. Why the `S` and what is the point of this simpler version?

Reply:
Same answer.